docs/samples/custom/torchserve/custom-server-with-external-storage.md

```diff
@@ -0,0 +1,32 @@
+# Torchserve with external storage
+
+For running torchserve with external storage, the model archive files and config.properties should be copied to the storage.
```

Comment:
I remember you had those instructions for how to copy models to the PVC, it can be useful here.